backend/src/main/java/com/jobwait/stat/Stats.java

```diff
@@ -0,0 +1,35 @@
+package com.jobwait.stat;
+
+import java.util.List;
+
+import org.springframework.http.HttpStatus;
+
+import com.jobwait.domain.Stat;
+import com.jobwait.fault.FaultException;
+import com.jobwait.stat.impl.JobTitleStat;
+
+public class Stats {
+    // touch this stuff
+    public static List<Stat> known = List.of(
+            new JobTitleStat("job-title"));
+
+    // don't touch this stuff
+    private static StatRefreshOfficer statRefreshOfficer = new StatRefreshOfficer();
+
+    public static Stat statFromId(String statId) {
+        List<Stat> matches = known.stream().filter(s -> s.id.equals(statId)).toList();
+        if (matches.size() == 0) {
+            throw new UnknownStatId(statId);
+        }
+        Stat stat = matches.getFirst();
```

Comment:
is there no `getOrElse` or `getFirstOrElse` ?

Reply:
rare java W
91a8538460ba624b1112c988479ef46d61dafa53